style.css

```diff
@@ -0,0 +1,22 @@
+body{
+    display: flex;
+    justify-content: flex-end;
+}
+nav{
+    
+}
+
+code{
+    background-color:lightgray
+}
+h1{
+    font-size:35px;
+    font-weight: lighter;
+}
+main{
+    
+    width:70%;
+    padding: 0px 60px 0px 20px;
+    margin: 50px;
+    
```

Comment:
Please remove all empty lines within the CSS elements, as it makes your code cleaner